cypress/support/cypress-commands/commands.js

```diff
@@ -186,7 +186,7 @@ Cypress.Commands.add('getDeviceVersion', () => {
     param: {},
     action: CONSTANTS.ACTION_CORE.toLowerCase(),
   };
-
+  cy.log('Device version intent: ' + JSON.stringify(requestMap));
   cy.sendMessagetoPlatforms(requestMap).then((response) => {
```

Comment:
This is a sendMessagetoPlatforms request and not an intent. It can have similar logs to 1st party app